plugins/VstBase/RemoteVstPlugin.cpp

```diff
@@ -1005,13 +1006,14 @@ bool RemoteVstPlugin::load( const std::string & _plugin_file )
 		return false;
 	}
 
+	const char id[5] = {
+		'0' + static_cast<char>(m_plugin->uniqueID >> 24),
+		'0' + static_cast<char>(m_plugin->uniqueID >> 16),
+		'0' + static_cast<char>(m_plugin->uniqueID >>  8),
+		'0' + static_cast<char>(m_plugin->uniqueID      ),
```

Comment:
```suggestion
		static_cast<char>(m_plugin->uniqueID >> 24),
		static_cast<char>(m_plugin->uniqueID >> 16),
		static_cast<char>(m_plugin->uniqueID >>  8),
		static_cast<char>(m_plugin->uniqueID      ),
```

Reply:
Resolving this as I have reverted commit cf3dbda